src/object/enemy_blocker.cpp

```diff
@@ -0,0 +1,44 @@
+//  SuperTux
+//  Copyright (C) 2007 Kovago Zoltan <DirtY.iCE.hu@gmail.com>
```

Comment:
There's something wrong with this copyright (same for the header). Please correct that.

Additionally, I'd recommend dedicate such small changes to [Public Domain](https://en.wikipedia.org/wiki/Public_domain) or using a license such as [CC0](https://creativecommons.org/about/cc0/) if you didn't steal the changes from other files (if that's the case, mark the original authors and use GPL, please).